apps/src/blockTooltips/DropletBlockTooltipManager.js

```diff
@@ -0,0 +1,105 @@
+var DropletFunctionTooltip = require('./DropletFunctionTooltip');
+var DropletFunctionTooltipMarkup = require('./DropletFunctionTooltip.html.ejs');
+
+/**
+ * @fileoverview Displays tooltips for Droplet blocks
+ */
+
+/**
+ * Handles displaying tooltips on Droplet blocks
+ * @param {DropletTooltipManager} dropletTooltipManager
+ * @constructor
+ */
+var DropletBlockTooltipManager = function (dropletTooltipManager) {
+  this.dropletTooltipManager = dropletTooltipManager;
+};
+
+var DEFAULT_TOOLTIP_CONFIG = {
+  interactive: true,
+  speed: 150,
+  maxWidth: 450,
+  position: 'right',
+  contentAsHTML: true,
+  theme: 'droplet-block-tooltipster',
+  offsetY: 2,
+  delay: 400
+};
+
+/**
+ * @param {Editor} dropletEditor
+ */
+DropletBlockTooltipManager.prototype.registerHandlers = function (dropletEditor) {
+  this.installTooltipsForCurrentCategoryBlocks(dropletEditor);
+  this.hideTooltipsOnBlockPick_(dropletEditor);
+
+  dropletEditor.on('changepalette', this.installTooltipsForCurrentCategoryBlocks.bind(this));
+  dropletEditor.on('toggledone', this.installTooltipsIfNotInstalled.bind(this));
+};
+
+DropletBlockTooltipManager.prototype.installTooltipsIfNotInstalled = function () {
+  if (!window.$) {
+    return; // TODO(bjordan): remove when $ available on dev server
+  }
+
+  if (!$('.droplet-hover-div').hasClass('tooltipstered')) {
+    this.installTooltipsForCurrentCategoryBlocks();
+  }
+};
+
+DropletBlockTooltipManager.prototype.installTooltipsForCurrentCategoryBlocks = function () {
+  if (!window.$) {
+    return; // TODO(bjordan): remove when $ available on dev server
+  }
+
+  var self = this;
```

Comment:
seems like everywhere else you've done .bind instead of self = this. Any reason not to follow suit here?